### PR TITLE
pom.xml: adding suuport for AARCH64 architecture in Netty/Transport/N…

### PR DESCRIPTION
**Pull request by ossdev07**
…ative/Epoll

AARCH 64 architecture support was missing in the exiting Netty-epoll code.
Now the Netty/Transport/Native/Epoll project can be used in aarch64 platform as well.

Motivation: 
To provide the support for AARCH64 architecture

Explain here the context, and why you're making that change.
What is the problem you're trying to solve.
Existing source code was not providing the support for AARCH64 arch. Not we have modified the pom.xml file and it is working fine.
Modification:
Modified the pom.xml file as attached in pull request.

Describe the modifications you've done.

Result:
It is working fine on AARCH64 architecture.

Fixes #<GitHub issue number>. 

If there is no issue then describe the changes introduced by this PR.

### CODE REVIEW

**Comment by normanmaurer** [comment]
@ossdev07 hmm can you update the commit message as I think its not about using the epoll transport but about building it or I am missing anything ?

**Comment by ossdev07** [comment]
commit message has been modified now.

File: pom.xml [hunk 1]
```diff
@@ -637,10 +637,10 @@
                 </requireMavenVersion>
                 <requireProperty>
                   <regexMessage>
-                    x86_64 JDK must be used.
+                    x86_64/AARCH64 JDK must be used.
                   </regexMessage>
                   <property>os.detected.arch</property>
-                  <regex>^x86_64$</regex>
+                  <regex>^(x86_64)|(aarch_64)$</regex>
```
[the remark this answers is not in the document]
Comment (normanmaurer) [reply pom.xml]: @ossdev07 can you also address @johnou s comment ? After thats done I will merge it

[the remark this answers is not in the document]
Comment (ossdev07) [reply pom.xml]: @normanmaurer I have updated the pom.xml as per @johnou suggestions. Please review it.

**Comment by normanmaurer** [comment]
@ossdev07 merged ... thanks

**Comment by ossdev07** [comment]
@normanmaurer can you please confirm if this package is released for aarch64 platform also

**Comment by ossdev07** [comment]
@normanmaurer can we add a profile in pom.xml to use cross-compiler for building  native code for aarch64, im asking it because nar-maven plugin is not working properly and to run it there should be that compiler in travis-CI

**Comment by normanmaurer** [comment]
@ossdev07 if you provide a pr for this I am happy to review.

**Pull request merged**